src/Interpreters/S3QueueLog.cpp

```diff
@@ -28,7 +28,9 @@ ColumnsDescription S3QueueLogElement::getColumnsDescription()
         {"hostname", std::make_shared<DataTypeLowCardinality>(std::make_shared<DataTypeString>())},
         {"event_date", std::make_shared<DataTypeDate>()},
         {"event_time", std::make_shared<DataTypeDateTime>()},
-        {"table_uuid", std::make_shared<DataTypeString>()},
+        {"database", std::make_shared<DataTypeString>()},
```

Comment:
Will we just will rename old table to `s3queue_log_0` on upgrade?

Reply:
yes should be so